Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -2426,6 +2426,298 @@ def update_cluster_settings_command(client: BotoClient, args: Dict[str, Any]) ->
             )
 
 
+class Lambda:
+    service = AWSServices.LAMBDA
+
+    @staticmethod
+    def get_function_configuration_command(client: BotoClient, args: Dict[str, Any]):
+        """
+        Retrieves the configuration information for a Lambda function.
+
+        Args:
+            client (BotoClient): The boto3 client for Lambda service
+            args (Dict[str, Any]): Command arguments including function name and optional qualifier
+
+        Returns:
+            CommandResults: Results of the operation with function configuration details
+        """
+        # Prepare parameters
+        function_name = args.get("function_name")
+        params = {"FunctionName": function_name}
+        if qualifier := args.get("qualifier"):
+            params["Qualifier"] = qualifier
+
+        # Get function configuration
+        response = client.get_function_configuration(**params)
+
+        # Remove ResponseMetadata for cleaner output
+        if "ResponseMetadata" in response:
+            del response["ResponseMetadata"]
```

Comment:
Why deleting `ResponseMetadata`? In all the other commands it is being used for verifying the status code.

Reply:
We don't want to add it to the context data.